packages/components/src/components/step/step.ts

```diff
@@ -105,12 +105,13 @@ export default class SdStep extends SolidElement {
 
   render() {
     const isLink = this.isLink();
-    const tag = this.notInteractive ? literal`div` : isLink ? literal`a` : literal`button`;
+    const tag = this.notInteractive || this.disabled ? literal`div` : isLink ? literal`a` : literal`button`;
```

Comment:
From our last conversation we said this would be the way to go:

	1.	 "current" interactive → no tabindex=-1 and button, therefore focusable and clickable
	2.	disabled → disabled button, therefore not focusable and clickable (addition: including aria-disabled=true)

Why did you now make it a div?

Reply:
Hey @mariohamann, I made it a div in order to be able to avoid disabled buttons as the ticket resquested but making the content readable and accessible for screen readers. I can revert it and make it a disabled button.

Reply:
Just for the moment, as @smfonseca will find a solution for ALL interactive components with disabled state.

Reply:
Updated